src/seedu/addressbook/storage/AddressBookDecoder.java

```diff
@@ -11,12 +11,7 @@
 
 import seedu.addressbook.data.AddressBook;
 import seedu.addressbook.data.exception.IllegalValueException;
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.*;
```

Comment:
According to the [Google Style Guide which we are using for this course](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports), wildcard imports is not used.